types/process.go

```diff
@@ -7,14 +7,39 @@ import (
 	"github.com/ethereum/go-ethereum/common"
 )
 
+type ProcessStatus uint8
```

Comment:
Make a search on the code and see if we are using raw uint8 in more places. Would be nice to replace all of them.